helper/bench-toolset.helper.bash/bench-toolset.bash

```diff
@@ -22,20 +24,19 @@ function build_bin()
 	(
 		cd "${dir}"
 		if [ -f "${bin_path}" ]; then
-			echo "[:)] found pre-built '${bin_path}' in build dir: '${dir}'" >&2
 			return
 		fi
 		${make_cmd} 1>&2
 		if [ ! -f "${bin_path}" ]; then
 			echo "[:(] can't build '${bin_path}' from build dir: '${dir}'" >&2
 			exit 1
 		fi
+		echo "[:)] build '${bin_path}' in build dir: '${dir}'" >&2
 	)
 	echo "${dir}/${bin_path}"
 }
 
 function build_bt()
 {
-	local dir="${1}"
-	build_bin "${dir}" 'bin/bench-toolset' 'make'
+	build_bin "${bt_repo_path}" 'bin/bench-toolset' 'make'
```

Comment:
We could init `bt_repo_path` inside build_bt, for that we got clean code and avoid global var.